README.md

```diff
@@ -83,7 +83,16 @@ From the repo:
       }'
    ```
 
-   If its the first time deploying the image then it should take few moments to download it.
+  If it's the first time deploying the image then it should take few moments to download it. You can also use the following variables to customize the sources of ovirt-flexvolume-driver and ovirt-volume-provisioner:
+   * FLEX_REGISTRY
+   * FLEX_REPOSITORY
+   * FLEX_VERSION
+   * PROVISIONER_REGISTRY
+   * PROVISIONER_REPOSITORY
+   * PROVISIONER_VERSION
+
+  You can either supply those variables to `docker -e` parameter or provide them as environment variables.
```

Comment:
This sentence is inaccurate,
You can pass in container environment variables:
 `podman run -e FLEX_REGISTRY=registry.example.com ... `
 or json variable for the ansible invocation:
`podman run .... provision '{"admin_username": .... , "flex_registry": "registry.example.com"}'